test/e2e/utils/bundles.go

```diff
@@ -0,0 +1,164 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"github.com/containers/buildah"
+	"github.com/containers/buildah/pkg/blobcache"
+	ocilayout "github.com/containers/image/v5/oci/layout"
+	"github.com/containers/image/v5/types"
+	"github.com/containers/storage"
+	"github.com/containers/storage/pkg/unshare"
+	"os"
+	"os/exec"
+	"path"
+	"sort"
+	"strings"
+	"time"
+)
+
+type Bundle struct {
```

Comment:
nit: I think it's potentially fine to define a one-off bundle type for our purposes here, but we can also use the `api.Bundle` type that is already used widely within OLM to define a bundle. If that works then I recommend we use that

Reply:
api.Bundle is missing a few fields that we need, mainly the default channel and the upload url. We can generate a url from the bundle name and a random string but default channel information is bundle specific.